client/eth.go

```diff
@@ -50,15 +47,35 @@ func (c *EthClient) TxPoolContent(ctx context.Context) (*TxPoolContent, error) {
 	return content, err
 }
 
-// TraceTransaction returns a transaction trace
 func (c *EthClient) TraceTransaction(ctx context.Context, hash string) (*Call, []*FlatCall, error) {
-	result := &Call{}
+	var result *Call
```

Comment:
What's the difference between these two ways to init in Golang?

Reply:
This won't work @dhrubabasu. This just creates an unallocated pointer (is `nil` not `&Call{}`)

Reply:
Addressed in a9525f4

Reply:
should do `var result Call`